bundle/manifests/patterns-operator.clusterserviceversion.yaml

```diff
@@ -25,7 +26,7 @@ metadata:
     operatorframework.io/suggested-namespace: patterns-operator-system
     operators.operatorframework.io/builder: operator-sdk-v1.19.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
-    repository: https://github.com/hybrid-cloud-patterns/patterns-operator
+    repository: https://github.com/hybridcloudpatterns/patterns-operator
```

Comment:
urL is wrong isn't it?

Reply:
blah brainfart. I meant to fix up only the quay repo url